src/test/mir-opt/inline/caller-with-trivial-bound.rs

```diff
@@ -1,5 +1,5 @@
+// ignore-wasm32-bare compiled with panic=abort by default
```

Comment:
wasm32-wasi is also compiled with panic=abort. Maybe make this

```suggestion
// ignore-wasm32 compiled with panic=abort by default
```